### ISSUE TITLE
A11y_.NETCore_ASP.NETCore_login_Use A Local Account to Login_Label or instructions: No visual instructions are provided for the sighted users to identify the input fields are required fields.

### ISSUE THREAD

**Issue by kupatkar99** [dotnet/aspnetcore#51153]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### Github Tags:
#A11ySev2; #A11yTCS; #DesktopWeb; #SH_.NETCore_Web_Oct2023; #.NET Core; #WCAG3.3.2; #Win11; #Labels and Instructions; #ChromiumEdge; 

### Environment Details:
App name: .NET Core
URL: http://localhost:5045/Account/Login
Chromium Edge: Version 119.0.2132.0 (Official build) dev (64-bit)

### Actual Result:
No visual instructions are provided for the sighted users to identify the input fields are required fields.

### User Impact:
Sighted users will not be able to know that the field is marked as required if there is no visual indication provided for the input fields.

### Expected Behavior

There should be visual instructions, indication or "Asterisk(*)" provided for the sighted users to identify the "Email", "Password" input fields are required fields.

### Steps To Reproduce

1. Open command prompt and enter command "mkdir foldername" e.g. "mkdir BlazorWasm".
2. Move to the created directory using command "cd foldername".
3. Run command "dotnet new blazorwasm" and hit ENTER key.
4.Run command "dotnet run".
5. Copy the First URL link from Command prompt and paste it in URL section of Browser and hit ENTER key.
6. TAB to "Login" control from left navigating and press ENTER.
7. "Login" screen will get open.
8. TAB to "Use A Local Account to Login" section.
9. TAB till "Email" input field and Observe that visual instructions are provided for the sighted users to identify the input fields are required fields or not.
<img width="959" alt="A11y_ NETCore_ASP NETCore_login_Use A Local Account to Login_Label or instructions" src="https://github.com/dotnet/aspnetcore/assets/98530069/d877dd96-4dc1-4e35-9df5-7ea4106af356">

**Comment by mkArtakMSFT** [comment]
#Closed;

**Comment by merriemcgaw** [comment]
Reopening on behalf of @kupatkar99. The issue is still present in the recent regression testing There are no visual instructions to let sighted users know fields are required before they try to submit the form. I will connect with the team and find out which MAS requirement that might fall under.

**Environment:**
SDK Version: 8.0.100-rtm.23519.30
Browser details: Microsoft Edge
Version 120.0.2165.1 (Official build) dev (64-bit)
![ASPNetCoreBug](https://github.com/dotnet/aspnetcore/assets/20996271/171c4291-daaa-44b4-a3dd-9f4987d05dab)

**Comment by rperez030** [comment]
In a traditional login form containing only username and password, there is no ambiguity as to which form field is required, therefore, at least from an accessibility perspective, the requirement is really not applicable since both are required. While I agree that this may be nice to have, I don't believe this should be considered a compliance failure, especially a blocking failure. I recommend this bug be closed or converted into a usable bug.

**Comment by mkArtakMSFT** [comment]
Thanks for the feedback @rperez030.
Closing as there is no accessibility bug here, and we don't plan to change this.
